src/web/app/pages-instructor/instructor-course-details-page/instructor-course-details-page.component.ts

```diff
@@ -190,16 +193,16 @@ export class InstructorCourseDetailsPageComponent implements OnInit {
     let blob: any;
 
     // Calling REST API only the first time to load the downloadable data
-    if (this.loading) {
+    if (this.isLoadingCsv) {
```

Comment:
I think we can just delete the if check. There is no need to have a variable to store the csv result.

This can just the normal flow where we set `isLoadingCsv` to `true` and then to `false`